### PR TITLE
cloud/aws: migrated ec2, ecs, rds and ssm client to sdk v2

### PR DESCRIPTION
**Pull request by j4k4**

### CODE REVIEW

File: pkg/cloud/aws/resourcegroupstaggingapi/resources_test.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+func TestResourcesManager_GetResources(t *testing.T) {
+	ctx := context.Background()
+	logger := logMocks.NewLoggerMockedAll()
+
+	client := new(gosoResMocks.Client)
+	client.On("GetResources", ctx, mock.AnythingOfType("*resourcegroupstaggingapi.GetResourcesInput")).Return(&resourcegroupstaggingapi.GetResourcesOutput{
+		PaginationToken: nil,
+		ResourceTagMappingList: []types.ResourceTagMapping{{
+			ResourceARN: aws.String("arn:aws:sqs:region:accountId:applike-test-gosoline-queue-id"),
```
Comment (bogdanfinn) [inline pkg/cloud/aws/resourcegroupstaggingapi/resources_test.go:25]:
```suggestion
			ResourceARN: aws.String("arn:aws:sqs:region:accountId:justtrack-test-gosoline-queue-id"),
```

File: pkg/cloud/aws/resourcegroupstaggingapi/resources_test.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		}},
+	}, nil)
+
+	srv := gosoRes.NewServiceWithInterfaces(client, logger)
+	r, err := srv.GetResources(ctx, gosoRes.Filter{
+		ResourceFilter: nil,
+		TagFilter:      nil,
+	})
+
+	expected := []string{"arn:aws:sqs:region:accountId:applike-test-gosoline-queue-id"}
```
Comment (bogdanfinn) [inline pkg/cloud/aws/resourcegroupstaggingapi/resources_test.go:36]:
```suggestion
	expected := []string{"arn:aws:sqs:region:accountId:justtrack-test-gosoline-queue-id"}
```

**Review by bogdanfinn — COMMENTED** [review]
one minor thing

File: pkg/cloud/aws/resourcegroupstaggingapi/resources_test.go (hunk 1 above)
Comment (j4k4) [reply pkg/cloud/aws/resourcegroupstaggingapi/resources_test.go]: changed

File: pkg/cloud/aws/resourcegroupstaggingapi/resources_test.go (hunk 2 above)
Comment (j4k4) [reply pkg/cloud/aws/resourcegroupstaggingapi/resources_test.go]: changed

**Review by bogdanfinn — APPROVED** [review]

**Pull request merged**